storage/client/queries/queries.go

```diff
@@ -286,30 +286,67 @@ const (
 			ORDER BY id DESC`
 
 	ValidatorsData = `
+		WITH self_delegations AS (
+			SELECT delegatee AS address, SUM(shares) AS shares
+				FROM chain.delegations
+				WHERE delegator = delegatee
+				GROUP BY delegatee
+		),
+		delegators_count AS (
+			SELECT delegatee AS address, COUNT(*) AS count
+				FROM chain.delegations
+				GROUP BY delegatee
+		),
+		active_nodes AS (
```

Comment:
Maybe rename `active_` prefix or add a comment, since IMO the "active" term could be a bit ambiguous. Does it refer to nodes that are currently registered as a validator, or does it refer only to nodes that are actually validating current epoch (are actually active). 

Some background:
The network supports a limited number of validators (consensus parameter, currently 120 on mainnet). If there are more validator nodes registered, then the top 120 (by stake) are picked at the start of the epoch and are selected as being the validator for the full duration of the epoch.

Looking at the DB setup I believe that `chain.nodes` contains all currently registered nodes and nodes with `voting_power > 0` are the actually active validators.


So I believe this query currently returns all registered validator nodes. I think this is fine, and what we actually want to return, but maybe just rename the `active_nodes` or add a clarifying comment.

Reply:
Updated!

Reply:
Note: API still exposes `active` field on validators
```
        active:
          type: boolean
          description: Whether the entity has a node that is registered for being a validator, node is up to date, and has successfully registered itself. It may or may not be part of validator set.
```

Should it be renamed?

And frontend https://github.com/oasisprotocol/explorer/pull/1478/files#diff-3d672171124172cffa1ca415897063fa7c5fb37391d024b4febf7ab4e76406e2R20-R21 labels them Active + Waiting. Should we change

Reply:
Yep this is intentional; the old `active` field has been renamed to `in_validator_set`. Apologies for the messiness :\ 

I could change the api fields so that we have `active` and `is_eligible`? In this case `active==in_validator_set`, which matches the description on the frontend. `is_eligible` would then be true if the validator node is registered and running. Wdyt?

The changes in the linked PR look good, the Explorer should continue to show Active/Waiting/Inactive.

Reply:
I'd keep `in_validator_set` and `active`, since these make the most sense to me. Also frontend PR seems to be updated to work correctly with these two, so let's keep it as it is?

Reply:
Agreed, I'll leave as-is and we can revisit in a follow-up if necessary.